platform/core/src/services/index.js

```diff
@@ -0,0 +1,4 @@
+import ServicesManager from './ServicesManager.js';
+import createUiNotificationService from './uiNotificationService';
```

Comment:
I'm not sure if `uiNotificationService` should have the first letter capitalized or not. Factory functions are similar to classes, and I would normally capitalize a class.

Reply:
That was my feeling when I changed the service name to be capitalized. I'm used to capitalizing not only because its a class type but services, controllers, classes, managers... so i can distinguish methods from group of methods grouped by a single purpose.. what do you think?

Reply:
I agree with your thoughts here 👍